labelbox/schema/export_filters.py

```diff
@@ -11,7 +11,24 @@
 ISO_8061_FORMAT = "%Y-%m-%dT%H:%M:%S%z"
 
 
-class SharedExportFilters(TypedDict):
+class BaseExportFilters(TypedDict):
+    data_row_ids: Optional[List[str]]
+    """ Datarow ids to export
+    Only allows MAX_DATAROW_IDS_PER_EXPORT_V2 datarows
+    Example:
+    >>>   ["clgo3lyax0000veeezdbu3ws4", "clgo3lzjl0001veeer6y6z8zp", ...]
+
+    """
+
+    global_keys: Optional[List[str]]
+    """ Global keys to export
+    Only allows MAX_DATAROW_IDS_PER_EXPORT_V2 datarows
```

Comment:
Customers won't know what the value of this is. Let's point them to the document
```
Before:
Only allows MAX_DATAROW_IDS_PER_EXPORT_V2 datarows

After:
Please refer to https://docs.labelbox.com/docs/limits on the allowed limit of global_keys
```

Reply:
Also, please follow up with Lance on updating our documentation to include global_keys
https://docs.labelbox.com/docs/limits
We'll need a new entry under Exports, `Max entries for global_keys filter`